res/database/Default/user/cujo.xml

```diff
@@ -1,764 +1,876 @@
-				<en>Schwalltau - The sweetest temptation since there is jam - Now also with belladonna cherries</en>
-			</description>
-			<conditions min_audience="2.3435" min_image="0" target_group="0" pro_pressure_groups="0" contra_pressure_groups="0"/>
-			<data quality="14" repetitions="3" duration="2" profit="750" penalty="1125" fix_infomercial_profit="1"/>
-		</ad>
-	</allads>
+<?xml version="1.0" encoding="utf-8"?>
+<tvtdb>
+	<version value="3" comment="Cujos Daten" />
+	<allnews>
+		<!-- "Superman und Supergirl" -->
+		<news id="cujo-news-superkerl-01" type="0" thread_id="cujo-news-superkerl" creator="8827" created_by="Cujo">
+			<title>
+				<de>Superweib verliebt sich in Superkerl</de>
+				<en>Superwoman falls in love with Superguy</en>
+				<pl>Superwoman zakochuje się w Superguyu</pl>
+			</title>
+			<description>
+				<de>Die beiden weltbekannten Superhelden haben sich gefunden. Sie findet ihn super. Er sie auch. Beide: „Unser erstes Date war super“.</de>
+				<en>The world-famous superheroes have found each other. She thinks he's super. He feels the same. Both: "Our first date was super".</en>
+				<pl>Znani na całym świecie superbohaterowie odnaleźli się nawzajem. Ona uważa, że on jest super. On czuje to samo. Oboje: „Nasza pierwsza randka była super”.</pl>
+			</description>
+			<effects>
+				<!-- "morgen oder uebermorgen, 12-16 Uhr" -->
+				<effect trigger="happen" type="triggernews" time="2,1,2,12,16" news="cujo-news-superkerl-02" />
+			</effects>
+			<data genre="1" price="1.0" quality="10" flags="2"/>
+			<availability year_range_from="1985" year_range_to="1995" />
+		</news>
+		<news id="cujo-news-superkerl-02" type="2" thread_id="cujo-news-superkerl" creator="8827" created_by="Cujo">
+			<title>
+				<de>Superweib und Superkerl verloben sich</de>
+				<en>Superwoman and Superguy get engaged</en>
+				<pl>Superwoman i Superguy zaręczają się</pl>
+			</title>
+			<description>
+				<de>Superkerl kann sein Glück nicht fassen: „Sie küsst super.“</de>
+				<en>Superguy can't believe his luck: "Her kisses are super."</en>
+				<pl>Superguy nie może uwierzyć w swoje szczęście: „Jej pocałunki są super”.</pl>
+			</description>
+			<effects>
+				<!-- "morgen oder uebermorgen, 13-17 Uhr" -->
+				<effect trigger="happen" type="triggernews" time="2,1,2,13,17" news="cujo-news-superkerl-03" />
+			</effects>
+			<data genre="1" price="1.0" quality="30" />
+		</news>
+		<news id="cujo-news-superkerl-03" type="2" thread_id="cujo-news-superkerl" creator="8827" created_by="Cujo">
+			<title>
+				<de>Superkerl und Superweib haben geheiratet</de>
+				<en>Superguy and Superwoman got married</en>
+				<pl>Superguy i Superwoman wzięli ślub</pl>
+			</title>
+			<description>
+				<de>Superweib über Superkerl: „In der Hochzeitsnacht war er super.“</de>
+				<en>Superweib about Superguy: "On the wedding night he was super."</en> <!-- fault Superwoman not Superweib -->
```

Comment:
fix Superweib here and remove the comment

Reply:
Done